bin/out

```diff
@@ -73,11 +74,16 @@ fi
 # execute
 #
 
+cntxt="$source_context"
+if [[ -n "$params_context" ]] ; then
+  cntxt="$params_context"
+fi
+
 jq -c -n \
   --arg state "$params_state" \
   --arg target_url "$( echo "$target_url" | buildtpl )" \
   --arg description "$( cat $description_path )" \
-  --arg context "$source_context" \
+  --arg context "$(echo "$source_base_context/$cntxt" | buildtpl)" \
```

Comment:
This will help fix https://github.com/dpb587/github-status-resource/issues/8, right?

Reply:
Correct. It should not break status lookups via `check` as long as the user does not include any env vars in the resource source context.